qoqo-qryd/src/tweezer_devices.rs

```diff
@@ -1020,6 +1052,50 @@ impl TweezerMutableDeviceWrapper {
             )
             .map_err(|err| PyValueError::new_err(format!("{:}", err)))
     }
+
+    /// Set the allowed Tweezer shifts from a list of tweezers.
+    ///
+    /// The items in the rows give the allowed tweezers that qubit can be shifted into.
+    /// For a list 1,2,3 the qubit can be shifted into tweezer 1, into tweezer 2 if tweezer 1 is not occupied,
+    /// and into tweezer 3 if tweezer 1 and 2 are not occupied.
```

Comment:
We should rather say in the row a qubit from tweezer 1 can be shifted into 2 and into 3 if 2 is not occupied by a qubit.